lib/msal-browser/src/crypto/PkceGenerator.ts

```diff
@@ -25,6 +25,47 @@ const RANDOM_BYTE_ARR_LENGTH = 32;
  * This file defines APIs to generate PKCE codes and code verifiers.
  */
 
+let pkceCode: PkceCodes | undefined;
```

Comment:
Please don't use global variables for this, can we store them as private class members on StandardController?

Reply:
Made this a private member of StandardController